common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -527,48 +542,91 @@ div.video {
           }
         }
 
-        a.add-fullscreen {
-          @extend %video-button;
-          background: url(../images/fullscreen.png) center no-repeat;
-          border-left: none;
+        .size-quality-controls {
+          display: block;
           float: left;
-          padding: 0 11px;
-          width: 30px;
-        }
+          margin: 0;
+          padding: 0;
+          list-style: none;
+          border-right: 1px solid #666;
 
+          .size-control {
+            margin: 0;
+            height: 46px;
+            border: none;
+
+            .add-fullscreen {
+              @extend %video-button;
+              display: inline-block;
+              width: 24px;
+              padding: 0 11px;
+              border: none;
+              line-height: 46px;
+              text-align: center;
+            }
+          }
 
-        a.quality-control {
-          @extend %video-button;
-          background: url(../images/hd.png) center no-repeat;
-          border-left: none;
-          float: left;
-          padding: 0 11px;
-          width: 30px;
+          .quality-control {
+            margin: 0;
+            height: 46px;
 
-          &.active {
-            background-color: #F44;
-            color: #0ff;
-            text-decoration: none;
-          }
+            .quality-toggle-control {
+              @extend %video-button;
+              float: left;
+              width: 30px;
+              padding: 0 11px;
+              border: none;
+
+              &.active {
+                background-color: #F44;
+                color: #0ff;
+                text-decoration: none;
+              }
 
-          &.is-hidden {
-            display: none !important;
+              &.is-hidden {
+                display: none !important;
+              }
+            }
           }
         }
 
-        div.lang {
-          & > a.hide-subtitles {
+        .language-control {
+          display: block;
+          float: left;
+          height: 46px;
+          border-right: 1px solid #666;
+
+          .subtitles-control {
             @extend %video-button;
-            @include transition(none);
-            box-shadow: inset 1px 0 0 #555;
-            background: url('../images/cc.png') center no-repeat;
-            border-left: none;
-            border-right: none;
-            padding: 0 11px;
-            width: 30px;
+            float: left;
+            padding: 0;
+            width: 46px;
+            text-align: center;
+            border: none;
+            white-space: nowrap;
 
-            &.off {
-              opacity: 0.7;
+            &.show_transcript {
+
+              .icon {
+                color: $white;
+              }
+            }
+
+            &.hide_transcript {
+
+              .icon {
+                color: $gray-l1;
+              }
+            }
+
+            &.active {
+              background-color: #F44;
```

Comment:
The JS switches class names depending on the state. I'm working on a video player (complete) overhaul in another branch so I'll be sure and clean this up there and make it more stateful. For this branch, I think this will need to stay. Good catch though.